zebra/rt_netlink.c

```diff
@@ -147,22 +147,6 @@ static bool is_proto_nhg(uint32_t id, int type)
 	return false;
 }
 
-/* Is vni mcast group */
```

Comment:
Don't you think moving this function is it's own logical step and should be in it's own commit?